libr/include/r_anal.h

```diff
@@ -1980,6 +1981,8 @@ R_API RAnalClassErr r_anal_class_vtable_delete(RAnal *anal, const char *class_na
 R_API void r_anal_class_list(RAnal *anal, int mode);
 R_API void r_anal_class_list_bases(RAnal *anal, const char *class_name);
 R_API void r_anal_class_list_vtables(RAnal *anal, const char *class_name);
+R_API void r_anal_class_list_offset_vfunction(RAnal *anal, const char *class_name, size_t offset);
```

Comment:
The definition of this function is not part of this pr, is it?

Reply:
Yeah, you're right

Reply:
Removed prototype